### PR TITLE
Move to lab and use its built in test coverage.

### PR DESCRIPTION
**Pull request by motdotla**

### CODE REVIEW

**Comment by motdotla** [comment]
So I'd like to move to lab instead of mocha. It gives us code coverage for free.

Otherwise, I think this is a good pull request from @maxbeatty.

Thoughts, @jessefulton, @maxbeatty, @jcblw.

File: .eslintrc [hunk 1]
```diff
@@ -1,8 +0,0 @@
-{
-  "env": {
-    "node": true
```
Comment (maxbeatty) [inline .eslintrc]: does lab set their internal eslint to node env? if not, this turns on some rules and would be worth keeping

File: lib/main.js [hunk 2]
```diff
@@ -1,37 +1,38 @@
-"use strict";
+'use strict'
```
Comment (maxbeatty) [inline lib/main.js]: [`quotes`](http://eslint.org/docs/rules/quotes.html) would be a good rule to to add to `.eslintrc`

File: lib/main.js [hunk 3]
```diff
@@ -1,37 +1,38 @@
...
   /*
    * Main entry point into dotenv. Allows configuration before loading .env and .env.$NODE_ENV
-   * @param {Object} options - valid options: path (".env"), encoding ("utf8")
+   * @param {Object} options - valid options: path ('.env'), encoding ('utf8')
    * @returns {Boolean}
   */
-  config: function(options) {
-    var path = ".env",
-    encoding = "utf8";
+  config: function (options) {
```
Comment (maxbeatty) [inline lib/main.js]: [`space-before-function-parentheses`](http://eslint.org/docs/rules/space-before-function-parentheses.html) would be a good rule to add to `.eslintrc`

File: lib/main.js [hunk 4]
```diff
@@ -1,37 +1,38 @@
-"use strict";
+'use strict'
 
-var fs = require("fs");
+var fs = require('fs')
```
Comment (maxbeatty) [inline lib/main.js]: [`semi`](http://eslint.org/docs/rules/semi.html) would be a good rule to add to `.eslintrc`

File: package.json [hunk 5]
```diff
@@ -3,11 +3,8 @@
   "version": "1.0.0",
   "description": "Loads environment variables from .env file",
   "main": "lib/main.js",
-  "directories": {
-    "test": "test"
-  },
   "scripts": {
-    "test": "gulp ci"
+    "test": "lab test/* --coverage && standard"
```
Comment (maxbeatty) [inline package.json]: if we're using lab for its built-in coverage why not also use its built in linter?

Comment (jcblw) [inline package.json]: [> No configuration. Just drop it in. The easiest way to enforce consistent style in your module/project.](https://github.com/feross/standard#one-style-to-rule-them-all)

Comment (maxbeatty) [inline package.json]: what's got everyone sold on `standard`? no config?

beyond `eslint` being bundled, it also offers more rules, wider community support, and more [integrations](http://eslint.org/docs/integrations/). to use with `lab`, you just pass `-L` or `--lint`

Comment (motdotla) [inline package.json]: I like the opinionated-ness of `standard`. And aside from that the style is good.  

Particularly for an open source lib, it's one less moving part.

Comment (maxbeatty) [inline package.json]: One less dependency is one less moving part :wink: `</eslint-push>`

**Comment by motdotla** [comment]
Alright, going to roll with this.

**Pull request merged**